soundbay/data.py

```diff
@@ -263,7 +263,7 @@ def _get_audio(self, path_to_file, begin_time, end_time, label, channel=None):
             start_time = begin_time
         data, _ = sf.read(str(path_to_file), start=start_time,
                           stop=start_time + requested_seq_length)
-        if channel is not None:
+        if channel is not None and channel>1:
```

Comment:
don't we want to check the shape of data instead of the value of channel?
If I have a multichannel input I still want to be able to read data from channel 1.

Reply:
Sometimes you have another channel that is empty.